packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -235,10 +235,11 @@ export async function createNewUsersConnectToOrgIfExists({
         const usernameOrEmail = usernamesOrEmails[index];
         const { orgId, autoAccept } = connectionInfoMap[usernameOrEmail];
         const [emailUser, emailDomain] = usernameOrEmail.split("@");
-        const username =
-          emailDomain === autoAcceptEmailDomain
+        const username = input.isOrg
```

Comment:
For a non sub-team invite, we shouldn't set username during the invite as user is allowed to change the username during the signup.